src/current/_includes/cockroachcloud/cluster-create-console-required-role.md

```diff
@@ -0,0 +1,7 @@
+**Required Permissions**
+
+A user must have one of the following [organization roles]({% link cockroachcloud/authorization.md %}#organization-user-roles) in order to create a cluster in that organization:
+
+- the [Cluster Creator role]({% link cockroachcloud/authorization.md %}#cluster-creator).
+- The [Cluster Administrator role]({% link cockroachcloud/authorization.md %}#cluster-administrator)  scoped to the entire organization (Cluster creator can be assigned to a signle cluster, which does not allow the user to create new clusters).
```

Comment:
```suggestion
- The [Cluster Administrator role]({% link cockroachcloud/authorization.md %}#cluster-administrator)  scoped to the entire organization (Cluster Creator can be assigned to a single cluster, which does not allow the user to create new clusters).
```

Reply:
Should the part in parenthesis be tagged on to the top bullet point for Cluster Creator? I was not sure why it was appended to the admin role bullet point. (Sorry if I'm missing something!)